public/app/plugins/panel/timeseries/plugins/AnnotationsPlugin2.tsx

```diff
@@ -166,19 +165,28 @@ export const AnnotationsPlugin2 = ({
             ctx.strokeRect(x0, y0, x1 - x0, y1 - y0);
           }
         } else {
+          // if multiple regions, don't shade
+          // @todo toggle functionality, new annotation config option?
+
           let y0 = u.bbox.top;
           let y1 = y0 + u.bbox.height;
 
           ctx.lineWidth = 2;
           ctx.setLineDash([5, 5]);
 
+          // @todo Don't render the vertical lines if multi lane is enabled
+          //@todo Don't render shaded region if multi-lane is enabled
+          // skipping this loop should do it
+
           for (let i = 0; i < vals.time.length; i++) {
             let color = getColorByName(vals.color?.[i] || DEFAULT_ANNOTATION_COLOR_HEX8);
 
             let x0 = u.valToPos(vals.time[i], 'x', true);
             renderLine(ctx, y0, y1, x0, color);
 
-            if (vals.isRegion?.[i]) {
+            // If dataframe does not have end times, let's omit rendering the region for now
+            // @todo do we want to fix isRegion to render a point when we're missing timeEnd?
+            if (vals.isRegion?.[i] && vals.timeEnd?.[i]) {
```

Comment:
`vals.timeEnd[i]` was throwing a runtime error when the annotation query had undefined for some timeEnd values (i.e. timeEnd was past the `to` date), which was breaking all the annotations in the panel. We probably want to create an issue to address the desired functionality in this case and add some test coverage.